chapter8.adoc

```diff
@@ -0,0 +1,30 @@
+[[chapter8]]
+[[Smsdedbg]]
+== `Smsdedbg`: Supervisor Domain External Debug
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "External Debug for Supervisor Domain", id=Smsdedbg_img]
+image::Smsdedbg.png[]
+
+The <<MSDCFG>> CSR contains the `sdedbgen` bit that controls whether the
+current scheduled SD is external-debug enabled. This bit is context switched
+(along with rest of the `msdcfg`) per SD. The `medbgen` is an enable control
+for external debug for the M-mode driven by the debug module and is expected
+to be established by the RoT (following RISC-V Security Model recommendation
+SR_GEN_007 and SR_GEN_012). When privilege is `M`, the `medbgen` gates the
```

Comment:
We need to separate the concepts between Smmtt Spec and External debug security Spec. My suggestion would be to move the content after this line to External debug security Spec. The separation would be following: 
- Smmtt Spec: Definition of medbgen, sdedbgen, MSDCFG, Smsdedbg ISA Extension
- External debug security Spec: The rules about how external debug and trace behaves based on medbgen, sdedbgen.
I am open to your suggestion for the separation. My goal is to have single source of truth.

Reply:
agree with your proposal @gokhankaplayan - i have separated the paragraph that line onwards and marked as as Informative for this spec (to be moved into the external debug security spec.